environments/hca-prod/terraform/buckets.tf

```diff
@@ -252,5 +252,7 @@ resource google_service_account_iam_binding dataflow_runner_user_binding {
 
   service_account_id = module.hca_dataflow_account.id
   role               = "roles/iam.serviceAccountUser"
-  members            = ["serviceAccount:${module.hca_argo_runner_account.email}"]
-}
+  members = [
```

Comment:
This gets clobbered when dagster.tf:72 gets evaluated, which is why this permission shows up in every terraform plan.